roles/net/defaults/main.yaml

```diff
@@ -1,6 +1,7 @@
 ---
 # vim:ts=2:sw=2:et:ai:sts=2
 
+net__enable: false
```

Comment:
I don´t think this belongs here.. The role does not use this variable at all, it only controls the playbook behaviour. I am not sure how to do it in this context, but I feel this is not the right solution

Reply:
Yes, I think you're right.

Since I wanted like this playbook to be runnable with no configuration (just an inventory), I'll do for _net_ what I'm already doing for _resolv:_

```yaml
- name: Run bootstraping roles
  hosts: all
  vars:
    play_net_enable: '{{ net__enable | d(false) }}'
    play_resolv_enable: '{{ resolv__enable | d(false) }}'
```

Reply:
SGTM!